lib/howl/commands/app_commands.moon

```diff
@@ -302,36 +313,54 @@ command.register
 -----------------------------------------------------------------------
 
 launch_cmd = (working_directory, cmd) ->
-  shell = howl.sys.env.SHELL or '/bin/sh'
-  p = Process {
-    :cmd,
-    :shell,
-    read_stdout: true,
-    read_stderr: true,
-    working_directory: working_directory,
-  }
-
-  buffer = ProcessBuffer p
-  editor = app\add_buffer buffer
-  editor.cursor\eof!
-  buffer\pump!
+  dispatch.launch ->
```

Comment:
The behavior (as designed) is that the handler runs in the 'foreground'. In this case we want the command to exit but to run something else in the background. I think this should be explicit with dispatch.launch.

To elaborate - there's the concept of a running command being 'active' for the duration it is running and you can have a stack of running commands (or interactions). When exiting, the stack unwinds until it's clear and the command line gets hidden. In this case we want the command to exit and the command line to be hidden, but another task to run. Hope that made some sense but I can elaborate further if needed.